### PR TITLE
pull only last 90 days of messages by default

### PR DESCRIPTION
**Pull request by claudiordgz**
- Nuke `clone` command
- Pull only last 90 days of messages
- filter by company id optional for messages

### CODE REVIEW

**Comment by richardgcosgrove** [comment]
Combat is currently working on a new API for message threading. You should check with @kevinbandy and @okrand if this will impact that effort.

**Comment by claudiordgz** [comment]
Hey @richardgcosgrove thanks for commenting. Thinker is the tool we use for syncing rethinkdb clusters with each other. Primarily used for syncing prod with the prod clone on a nightly basis.

It wouldn't affect the feature unless you're testing against the prod clone (please don't do that, ever).

File: lib/commands/sync.js [hunk 1]
```diff
@@ -54,9 +54,31 @@ let HELPTEXT = `
     --indexesOnly                             Just create indexes
     --buffer                                  <number> - Minimum number of connections available in the pool, default 20
     --max                                     <number> - Maximum number of connections available in the pool, default 50
+    --allMessages                             Pulls all messages instead of last 3 months (default: false)
```
Comment (funkswing) [inline lib/commands/sync.js:57]: In the future we can add the ability to pass in the range of messages to sync.... B U T  O F   C O U R S E  T H A T ' S    N O T   I M P O R T A N T   N O W

**Review by funkswing — APPROVED** [review]
The (last 90 days of) Messages Must Flow

**Pull request merged**